### ISSUE TITLE
transform-related function names incorrectly marked as lints

### ISSUE THREAD

**Issue by lifeiscontent** [brigade/scss-lint#30]
when I do something like this

``` scss
.arrow {
  @include transform(scaleY(-1));
}
```

I get this scss-lint error `Names of variables, functions, and mixins should be lowercase and not contain underscores. Use hyphens instead.`
